src/components/UTIL_PageHeader.component

```diff
@@ -63,17 +63,17 @@
     <apex:attribute name="saveAndNewDisabled" type="Boolean" default="false"
         description="Whether the save and new button should be disabled, defaults to false."/>
 
-    <div class="slds-page-header" role="banner">
+    <div class="slds-page-header" role="banner" id="header">
         <div class="slds-grid">
             <div class="slds-col slds-col_padded">
                 <apex:outputPanel layout="none" rendered="{!showBreadcrumb}">
                     <nav role="navigation" class="slds-p-bottom_x-small">
                         <p id="bread-crumb-label" class="slds-assistive-text">{!$Label.lblYouAreHere}</p>
-                        <ol class="slds-breadcrumb slds-list_horizontal slds-wrap" aria-labelledby="bread-crumb-label">
-                            <li class="slds-breadcrumb__item slds-text-title_caps">
+                        <ol class="slds-breadcrumb slds-list_horizontal slds-wrap" aria-labelledby="bread-crumb-label" id="headerBreadcrumb">
+                            <li class="slds-breadcrumb__item slds-text-title_caps" id="parentEntity">
```

Comment:
Same with these Id's. It's possible that some other native Salesforce markup might use Id's with generic names such as these; or that pages that use this component might use these same Id's.